src/main/java/example/mod/client/plugin/ExampleJourneymapPlugin.java

```diff
@@ -145,7 +145,7 @@ void onMappingStarted(ClientEvent event)
         if (jmAPI.playerAccepts(ExampleMod.MODID, DisplayType.Marker))
         {
             BlockPos pos = Minecraft.getInstance().player.blockPosition();
-            SampleMarkerOverlayFactory.create(jmAPI, pos, 64, 256);
+            //SampleMarkerOverlayFactory.create(jmAPI, pos, 64, 256);
```

Comment:
This should not be commented out.

Reply:
It was commented out because if I didn't, the fullscreen map wouldn't open at all, and something went weird during startup that caused the vanilla hotbar to render at entirely the wrong scale.

Reply:
This has now been split off to #13